test/Python/python_runtime.swift

```diff
@@ -5,6 +5,7 @@
 
 import Python
 import StdlibUnittest
+import struct TensorFlow.TensorShape
```

Comment:
These tests should also move to `TensorFlowRuntime/numpy_conversion.swift`. Python tests should not depend on the `TensorFlow` module.